app/controllers/users/sessions_controller.rb

```diff
@@ -1,7 +1,10 @@
 class Users::SessionsController < Devise::SessionsController
-  after_action :verify_authorized, except: [:new, :create, :destroy]
+  after_action :verify_authorized, except: [:new, :create, :destroy, :logout]
 
   def after_sign_in_path_for(resource)
     resource.after_sign_in_path
   end
+
+  def logout
```

Comment:
@jedschneider devise doesn't look like it implements a view for logout... I'm going to leave this for now unless you have a resource to point me to.